### ISSUE TITLE
Full error from OpenAI: Connection error

### ISSUE THREAD

**Issue by syzygyred** [TomFrankly/pipedream-notion-voice-notes#51]
Hi,
I've tried to follow your instructions many times, deleting everything and restarting along the way. The audio file is in Dropbox, that part completes successfully; I have connected to my ChatGPT account, and I have entered my payment details there - I have generated a key as suggested, multiple times, but the Notion step wont complete. The Notion DB is successfully selected, so seems to be the ChatGPT step for some reason.
Cheers

An error occured while attempting to split the file into chunks, or while sending the chunks to OpenAI. If the full error below says "Unidentified connection error", please double-check that you have entered valid billing info in your OpenAI account. Afterward, generate a new API key and enter it in the OpenAI app here in Pipedream. Then, try running the workflow again. If that does not work, please open an issue at this workflow's Github repo: https://github.com/TomFrankly/pipedream-notion-voice-notes/issues Full error from OpenAI: Connection error.

DETAILS
    at Object.chunkFileAndTranscribe (file:///tmp/__pdg__/dist/code/4cf355a52ab0f9c275ba953eea42492276c4b796f961fdffefa87942b1ced4df/code/Notion-Voice-Notes.mjs:391:11)
    at Object.run (file:///tmp/__pdg__/dist/code/4cf355a52ab0f9c275ba953eea42492276c4b796f961fdffefa87942b1ced4df/code/Notion-Voice-Notes.mjs:1901:22)
    at null.executeComponent (/var/task/launch_worker.js:267:22)
    at MessagePort.messageHandler (/var/task/launch_worker.js:764:28)

**Comment by TomFrankly** [comment]
Hi @syzygyred,

Can you make sure your workflow is updated using these steps, then try again? https://thomasjfrank.com/how-to-transcribe-audio-to-text-with-chatgpt-and-notion/#update

If it doesn't work after that, it means OpenAI is having API problems.

**Comment by syzygyred** [comment]
Hi @TomFrankly ,

Appreciate the help on this. I have ensured this is updated to the latest version using your link/guide, but have tried many times and I'm not getting anywhere.

I only get the following error below.

I have a ChatGPT account with credit on it, have generated the API secret many times, following the instructions, but am all out of ideas now.

Cheers,
Steven


Error
Failed to download file: Response code 404 (Not Found)

DETAILS
    at Object.downloadToTmp (file:///tmp/__pdg__/dist/code/0f5c246b7952a2a4d8fb0ab5802d7166cb897a9065e92201a91a31993507bd44/code/Notion-Voice-Notes.mjs:321:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Object.run (file:///tmp/__pdg__/dist/code/0f5c246b7952a2a4d8fb0ab5802d7166cb897a9065e92201a91a31993507bd44/code/Notion-Voice-Notes.mjs:1949:5)
    at null.executeComponent (/var/task/launch_worker.js:267:22)
    at MessagePort.messageHandler (/var/task/launch_worker.js:764:28)

**Comment by syzygyred** [comment]
@TomFrankly thank you Thomas, all is resolved now. There were various issues afoot I think in the course of this!
1. I first tried this when you first released it, but I failed to get it working at that time and only returned to it now - so my ChatGPT free trial $5 credit had expired since that time, unaware to me (it wasn't clear if the ChatGPT would auto-charge me or if I need to add credit in advance, so I did the latter anyway).
3. You were absolutely correct - the test files I was using from Dropbox were older ones, and I was unclear that the selection I was making in the ChatGPT section (all in Test mode) were all on an older file.
Thanks a million!
